src/status_im/ui/components/desktop/tabs.cljs

```diff
@@ -38,9 +39,7 @@
 (defn tab [index content view-id active?]
   [react/touchable-highlight {:style    (merge tabs.styles/tab-container {:flex 1})
                               :disabled active?
-                              :on-press #(do
-                                           (re-frame/dispatch [:navigate-to :home])
```

Comment:
@siphiuel think this line is removed from my merge with some of your code. lmk if my misunderstanding is off here and we actually do need this for proper navigation!

Reply:
We don't want to persist right pane contents when navigating to a different tab. This was a problem with QR code PR, so I've added this event dispatch there.

Reply:
Ah ok, thank you clarifying this. And yes that makes total sense. Will add it back in. :+1: